layers/core_checks/render_pass_cc_validation.cpp

```diff
@@ -1478,20 +1478,20 @@ bool CoreChecks::ValidateRenderpassAttachmentUsage(RenderPassCreateVersion rp_ve
                         if (!enabled_features.linear_color_attachment_features.linearColorAttachment) {
                             vuid = use_rp2 ? "VUID-VkSubpassDescription2-pInputAttachments-02897"
                                            : "VUID-VkSubpassDescription-pInputAttachments-02647";
-                            skip |= LogError(
-                                device, vuid,
-                                "%s: Input attachment %s format (%s) does not contain VK_FORMAT_FEATURE_COLOR_ATTACHMENT_BIT "
-                                "| VK_FORMAT_FEATURE_DEPTH_STENCIL_ATTACHMENT_BIT.",
-                                function_name, error_type.c_str(), string_VkFormat(attachment_format));
+                            skip |= LogError(device, vuid,
+                                             "%s: Input attachment %s format (%s) does not contain any of "
+                                             "VK_FORMAT_FEATURE_COLOR_ATTACHMENT_BIT "
+                                             "or VK_FORMAT_FEATURE_DEPTH_STENCIL_ATTACHMENT_BIT.",
```

Comment:
Just my 2 cents. I think this would be easier to read. Since the VUID description gets amended to the error message.

```suggestion
                                             "%s: Input attachment %s format (%s) lacks sufficient format features!",
```

> %s: Input attachment %s format (%s) lacks sufficient format features! All attachments in pInputAttachments that are not VK_ATTACHMENT_UNUSED must have image formats whose [potential format features](https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#potential-format-features) contain at least VK_FORMAT_FEATURE_COLOR_ATTACHMENT_BIT or VK_FORMAT_FEATURE_DEPTH_STENCIL_ATTACHMENT_BIT

Reply:
Yes this is better. Not sure about the `!` though?